### PR TITLE
Feat: CMS 4.1 compatible

### PR DESCRIPTION
**Pull request by joshyu**

### CODE REVIEW

**Comment by fsbraun** [comment]
@joshyu Great work! I am also concerned with Django 5 compatibility. Did you take a look while at it?

File: djangocms_url_manager/models.py [hunk 1]
```diff
@@ -130,7 +129,9 @@ def get_content(self, show_draft_content=False):
 
 
 class UrlGrouper(AbstractUrlGrouper):
-    def get_content_queryset(self):
+    def get_content_queryset(self, show_draft_content=False):
+        if CMS_41 and hasattr(Url, "admin_manager") and show_draft_content:
```
Comment (fsbraun) [inline djangocms_url_manager/models.py:133]:
```suggestion
        if hasattr(Url, "admin_manager") and show_draft_content:
```
Just a suggestion: In django CMS 4 `Url` never has the attribute `admin_manager`, so the version check can go away.

Comment (joshyu) [reply djangocms_url_manager/models.py:133]: @fsbraun , You are right. I will remove this line.

**Comment by joshyu** [comment]
> @joshyu Great work! Look like it's working!
> 
> I am also concerned with Django 5 compatibility. Did you take a look while at it?

@fsbraun ,  sorry I have too much work for now. but I will check it later.

**Review by fsbraun — APPROVED** [review]

**Pull request merged**